CorsixTH/Lua/diseases/serious_radiation.lua

```diff
@@ -43,8 +43,18 @@ disease.initPatient = function(patient)
   patient:setLayer(1, math.random(0, 3) * 2)
   patient:setLayer(3, 0)
   patient:setLayer(4, 0)
+  patient.th:setPatientEffect(AnimationEffect.Glowing)
+end
+disease.treatPatient = function(patient)
+  patient.th:setPatientEffect(AnimationEffect.None)
+end
+disease.afterLoad = function(patient)
```

Comment:
This seems a little off in the way of doing an afterLoad that I've seen before. Especially in other files.
Not saying it's wrong in this case, just not what I normally see.